vm/isolated/isolated.go

```diff
@@ -39,6 +41,7 @@ type instance struct {
 	os          string
 	targetAddr  string
 	targetPort  int
+	USBAuth     string
```

Comment:
This is a private field and should start with a small letter: `usbAuth`.

Reply:
Also, I think it's better to pass USB device number here and then form `usbAuth` string in `repair()`.